pages/[[...permalink]].tsx

```diff
@@ -60,11 +64,20 @@ export const getStaticProps = (async (context) => {
   );
   return { props: { schema } };
 }) satisfies GetStaticProps<{
-  schema: any;
+  schema: IsomerPageSchema;
```

Comment:
question: this is a little complicated because we may have changes to the schema in the future, making JSON files created in the previous schema versions be incompatible with this typing.

Can I check whether the `satisfies` keyword would throw issues at build time, if the JSON file does not match the IsomerPageSchema? If yes, then we might not be able to use this typing here.

Reply:
nope, doesn't throw issues at build time, for e.g if I remove `layout` from the json, it still builds, but obviously at run time the site doesn't work

Reply:
Nice ok